recipes/glog/all/conanfile.py

```diff
@@ -18,83 +21,88 @@ class GlogConan(ConanFile):
+        if Version(self.version) >= "0.5.0":
+            tc.variables["WITH_PKGCONFIG"] = True
+            if self.settings.os == "Emscripten":
+                tc.variables["WITH_SYMBOLIZE"] = False
+                tc.variables["HAVE_SYSCALL_H"] = False
+                tc.variables["HAVE_SYS_SYSCALL_H"] = False
+            else:
+                tc.variables["WITH_SYMBOLIZE"] = True
+            tc.variables["WITH_UNWIND"] = self.options.get_safe("with_unwind", default=False)
+        tc.variables["BUILD_TESTING"] = False
+        tc.variables["WITH_GTEST"] = False
+        tc.generate()
 
     def _patch_sources(self):
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+        apply_conandata_patches(self)
         # do not force PIC
-        if tools.Version(self.version) <= "0.5.0":
-            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+        if Version(self.version) <= "0.5.0":
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
                                   "set_target_properties (glog PROPERTIES POSITION_INDEPENDENT_CODE ON)",
                                   "")
 
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["WITH_GFLAGS"] = self.options.with_gflags
-        self._cmake.definitions["WITH_THREADS"] = self.options.with_threads
-        if tools.Version(self.version) >= "0.5.0":
-            self._cmake.definitions["WITH_PKGCONFIG"] = True
-            if self.settings.os == "Emscripten":
-                self._cmake.definitions["WITH_SYMBOLIZE"] = False
-                self._cmake.definitions["HAVE_SYSCALL_H"] = False
-                self._cmake.definitions["HAVE_SYS_SYSCALL_H"] = False
-            else:
-                self._cmake.definitions["WITH_SYMBOLIZE"] = True
-            self._cmake.definitions["WITH_UNWIND"] = True
-        self._cmake.definitions["BUILD_TESTING"] = False
-        self._cmake.configure()
-        return self._cmake
-
     def build(self):
         self._patch_sources()
-        cmake = self._configure_cmake()
+        cmake = CMake(self)
+        cmake.configure()
         cmake.build()
 
     def package(self):
-        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
-        cmake = self._configure_cmake()
+        self.copy("COPYING", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
```

Comment:
for some reason this is not enough?
```

[HOOK - conan-center.py] post_package(): ERROR: [DEFAULT PACKAGE LAYOUT (KB-H013)] Unknown folder 'share' in the package (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H013) 
```
I dont see it locally in my package folder...